src/tests/unit/time/UtcTimeTest.php

```diff
@@ -93,12 +93,12 @@ public function testGetSecondsSinceDateTime()
             $actual = UtcTime::getSecondsSinceDateTime($testCase['value']);
             
             // Assert:
-            $this->assertEquals($testCase['expected'], $actual, sprintf(
-                'Expected %s to result in %s, not %s.',
+            $this->assertEqualsWithDelta($testCase['expected'], $actual, 1,
+                sprintf('Expected %s to result in %s, not %s.',
                 var_export($testCase['value'], true),
```

Comment:
Another PR, but curious, why are these wrapped with var_export()? PHP is forgiving on data types in sprintf's. $numericValue can be sprintf'd in a %s.

Reply:
I think maybe because they are time values, but you can ask Matt about it, if you want.  It looks like he's the originator of them.